californium-core/src/main/java/org/eclipse/californium/core/CoapServer.java

```diff
@@ -147,7 +147,7 @@ public CoapServer(final NetworkConfig config, final int... ports) {
 		}
 		
 		// resources
-		this.root = createRoot();
+		this.root = new RootResource();
```

Comment:
Right now it is very urgent for us to fix it. We can still live with it. For now we can park this until we find a better solution for this.